src/workspaces.c

```diff
@@ -333,6 +333,8 @@ workspaces_find(struct workspace *anchor, const char *name, bool wrap)
 				return target;
 			}
 		}
+	} else if (!strcasecmp(name, "current")) {
+		return anchor;
```

Comment:
This seems like accidentally committed. If you send a new PR with only this change (and updated docs for `SendToDesktop`) I'd be fine merging this if @johanmalm agrees.

It's listed in the scope repo as `4.7.1.4` for `GoToDesktop` and cat `C` and it really doesn't make sense for that.
For `SendToDesktop` in combination with this PR however, it pretty much makes sense IMHO.

Reply:
This is basically to test my use case. I'll drop it after WIP.

Reply:
Dropped.